addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -185,6 +175,17 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeActivityBoxView() {
+            if (this.thread && this.thread.activities.length > 0) {
```

Comment:
Missing condition on `hasActivities` that was present on the template.

This is important because some rare type of threads don't have the activity mixin.